omegatrade-app/frontend/src/app/components/company/update-company/update-company.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { FormBuilder, Validators } from '@angular/forms';
+import { RestService } from '../../../services/rest.service';
+import { SnackBarService } from '../../../services/snackbar.service';
+
+@Component({
+  selector: 'app-update-company',
+  templateUrl: './update-company.component.html',
+  styleUrls: ['./update-company.component.css']
+})
+export class UpdateCompanyComponent implements OnInit {
+  companyForm: any;
+  loader: boolean = false;
+  constructor(private snackBarService: SnackBarService, private restService: RestService, private formBuilder: FormBuilder, private dialogRef: MatDialogRef<UpdateCompanyComponent>, @Inject(MAT_DIALOG_DATA) public data: any) {
+    this.companyForm = this.formBuilder.group({
+      companyId: ['', []],
+      companyName: ['', [Validators.required]],
+      companyShortCode: ['', [Validators.required]],
+      created_at: ['']
```

Comment:
Are we asking users to add **created_at** in the form? if not we should not pass this to backend

Reply:
this issue has been resolved.